scripts/upgrade

```diff
@@ -155,4 +153,16 @@ systemctl reload nginx
 # REMOVE BACKUP_CORE_ONLY
 #=================================================
 
+backup_core_only=$(ynh_app_setting_get $app backup_core_only)
+# If "backup_core_only" is still set in settings.yml, print a warning and run the upgrade as usual.
+# Setting is deleted at the end of the script to avoid side effect ("ynh_backup_before_upgrade" maybe).
+if [ -z $backup_core_only ]
```

Comment:
Not `-z`, or you'll get the message only if backup_core_only isn't set.
Use instead `"$backup_core_only" == "1"`